packages/flutter/test/widgets/draggable_scrollable_sheet_test.dart

```diff
@@ -260,5 +264,48 @@ void main() {
 
       debugDefaultTargetPlatformOverride = null;
     });
+
+    testWidgets('ScrollNotification correctly dispatched when flung without covering its container', (WidgetTester tester) async {
+      final List<Type> notificationTypes = <Type>[];
+      await tester.pumpWidget(_boilerplate(
+        null,
+        onScrollNotification: (ScrollNotification notification) {
+          notificationTypes.add(notification.runtimeType);
+          return false;
+        },
+      ));
+
+      await tester.fling(find.text('Item 1'), const Offset(0, -200), 200);
+      await tester.pumpAndSettle();
+
+      final List<Type> types = <Type>[
+        ScrollStartNotification,
+        ScrollEndNotification,
```

Comment:
@gspencergoog Do you think `UserScrollNotification` and `ScrollUpdateNotification` should also be dispatched while dragging without covering its container?

Reply:
Per the docs on https://github.com/flutter/flutter/blob/master/packages/flutter/lib/src/widgets/scroll_notification.dart#L43, it seems like there should be.

Reply:
That said, it seems like in this case we could file a seaprate bug for that and fix it in a separate PR.

Reply:
(With a TODO here for it)

Reply:
I added TODO comment. And I will send another PR for this bug. Do I need to post issue before sending PR?

Reply:
Issuebis helpful to make sure we don't lose track